src/index.html

```diff
@@ -3,11 +3,54 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
```

Comment:
1. stars--0 is modificator, so in one element you should have `class=stars stars--0` 
2. 1 div, which now has class 'stars' is not necessary, but if you want to have some container it can be just div without any class (or with class 'wrapper'  for example)
3. make empty lines beetween blocks